tests/plugins_test.go

```diff
@@ -61,6 +61,8 @@ exec docker run --log-driver=none --rm -i {{.ImageName}}:{{.Version}} "$@"
 	imageOverrides = map[string][]string{
 		// betterproto (at least at v1.2.5) doesn't support eliza since it uses client streaming
 		"buf.build/community/danielgtaylor-betterproto": {"petapis"},
+		// pluginrpc-go doesn't support eliza since it has streaming RPCs
+		"buf.build/pluginrpc/pluginrpc-go": {"petapis"},
```

Comment:
Maybe there could be a plugin option to silently ignore streaming RPCs instead of failing here:

https://github.com/pluginrpc/pluginrpc-go/blob/c5376dd4649727825f9d1dd2045c53fb5e71b24f/cmd/protoc-gen-pluginrpc-go/main.go#L84-L90

That way you could still generate the module with the plugin, skipping over unsupported RPCs silently.

Reply:
This is done in v0.3.0